src/pages/HistoryPage/EpochPanel.tsx

```diff
@@ -92,86 +101,140 @@ export const EpochPanel = ({
-              <Text bold font="inter" size="large">
-                {sent.filter(g => g.gift_private?.note).length}
-              </Text>
-            </Box>
-            <Box>
-              <Text variant="label">Received</Text>
-              <Text bold font="inter" size="large">
-                {received.filter(g => g.gift_private?.note).length}
-              </Text>
-            </Box>
-          </Box>
-          <button onClick={() => setShowLess(false)}>
-            <Text
-              variant="label"
-              css={{ color: '$secondaryText', cursor: 'pointer' }}
-            >
-              Show More
-            </Text>
-          </button>
-        </Panel>
-      ) : (
-        <Panel nested>
-          <Box
-            css={{
-              display: 'flex',
-              gap: '$md',
-              justifyContent: 'space-between',
-              alignItems: 'flex-start',
-            }}
-          >
-            <Box css={{ display: 'flex', gap: '$sm', mb: '$xs' }}>
-              {['Received', 'Sent'].map((label, index) => (
+          <Flex column css={{ gap: '$sm' }}>
+            <Text>Notes</Text>
+            <Flex css={{ gap: '$md' }}>
+              <Box css={{ display: 'flex', gap: '$sm', mb: '$xs' }}>
                 <Button
-                  key={label}
                   outlined
                   size="small"
                   css={{ borderRadius: '$pill' }}
-                  onClick={() => setTab(index)}
+                  onClick={() => setTab(0)}
                 >
                   <Text
                     variant="label"
-                    css={{ color: tab === index ? '$text' : '$secondaryText' }}
+                    css={{
+                      color: tab === 0 ? '$text' : '$secondaryText',
+                    }}
                   >
-                    {label}
+                    {received.filter(g => g.gift_private?.note).length} Received
                   </Text>
                 </Button>
-              ))}
-            </Box>
-            <button
-              onClick={event => (setShowLess(true), event.stopPropagation())}
-            >
-              <Text
-                variant="label"
-                css={{ color: '$secondaryText', cursor: 'pointer' }}
+                <Button
+                  outlined
+                  size="small"
+                  css={{ borderRadius: '$pill' }}
+                  onClick={() => setTab(1)}
+                >
+                  <Text
+                    variant="label"
+                    css={{
+                      color: tab === 1 ? '$text' : '$secondaryText',
+                    }}
+                  >
+                    {sent.filter(g => g.gift_private?.note).length} Sent
+                  </Text>
+                </Button>
+              </Box>
+            </Flex>
+          </Flex>
+          {isAdmin && (
+            <Flex column css={{ gap: '$sm' }}>
+              <Text>Distribution</Text>
+              <Button
```

Comment:
This should link to the distributions page instead when the values feature is enabled, like the [previous table](https://github.com/coordinape/coordinape/blob/eb9f47dbe0/src/pages/AdminPage/components.tsx#L516)

Reply:
I discussed this with @spydermonkee  and we agreed to download the CSV from here until this feature is implemented there

Reply:
@levity  you will find the note here https://www.figma.com/file/L4oZwYVUdpgacZtwipGaYe/App-1.5?node-id=12024%3A28366
I don't mind changing it if that is required now

Reply:
I see. Yes, please change it so that it checks if the vaults feature is enabled, and shows Distributions if true, and Export CSV if not. We need a link to that distributions page while we're testing vaults. cc @spydermonkee

Reply:
Done